src/services/business-lookup-services.ts

```diff
@@ -32,10 +32,9 @@ export default class BusinessLookupServices {
    * @param query code or words to search
    * @returns a promise to return the search results
    */
-  static async search (query: string): Promise<BusinessLookupResultIF[]> {
-    const legalType = 'BC,A,ULC,C,S,XP,GP,LP,CUL,XS,LLC,LL,BEN,CP,CC,XL,FI,XCP,PA'
+  static async search (query: string, searchStatus: string, legalTypes: string): Promise<BusinessLookupResultIF[]> {
```

Comment:
I see you're passing in 2 new formal parameters, but:
- you're using only 1 of them
- is the other code that calls this updated to pass both new params?

Reply:
You're right Sev, good catch.

Basically, this is what it was before in the shared components: https://github.com/bcgov/bcrs-shared-components/blob/4c62c41230422c5ceef744176fdc1bb04d6b5372/src/components/business-lookup/BusinessLookup.vue#L227

I noticed the searchStatus was missing all this time. It was there in the shared components repo but I noticed it's not being used in here. I was debating whether to remove it or not but I decided to keep it.

Let me know if you would rather me delete it from the business lookup is the shared components repo.

Reply:
Yeah, it's a bit tricky. I bet that second formal parameter was added since the original code in here was written, and the original code possibly uses an older version of BusinessLookup that didn't have that second parameter. Anyway, it needs to be untangled... Good luck and lmk if I can help.

Reply:
So Sev, here's what I found out. The `search` function in the business lookup services is different between UIs: https://github.com/bcgov/business-edit-ui/blob/21e095886fe05379e58842bb04cac6c47526c331/src/services/business-lookup-services.ts#L36

The search status is used in Edit UI but not in here. So, I can't remove it from the lookup in the shared repo (I'm glad I didn't do that in my last PR there).

So, I don't see any other way other than to specify the parameter and not use it to bypass the issue. What do you think?

Reply:
Sure. Just leave a code comment explaining it.

Please update the params in the JSDoc while you're there. Thx!

Reply:
Perfect. Resolving.